Sources/BraveWallet/Crypto/Asset Details/AssetDetailView.swift

```diff
@@ -36,173 +36,226 @@ struct AssetDetailView: View {
-    Section(
-      header: WalletListHeaderView(title: Text(Strings.Wallet.transactionsTitle))
-    ) {
+    
+  private func accontBalanceRow(_ viewModel: AccountAssetViewModel) -> some View {
+    HStack {
+      AddressView(address: viewModel.account.address) {
+        AccountView(address: viewModel.account.address, name: viewModel.account.name)
+      }
+      let showFiatPlaceholder = viewModel.fiatBalance.isEmpty && assetDetailStore.isLoadingPrice
+      let showBalancePlaceholder = viewModel.balance.isEmpty && assetDetailStore.isLoadingAccountBalances
       Group {
-        if assetDetailStore.transactionSummaries.isEmpty {
-          Text(Strings.Wallet.noTransactions)
-            .font(.footnote)
-        } else {
-          ForEach(assetDetailStore.transactionSummaries) { txSummary in
-            Button(action: {
-              self.transactionDetails = assetDetailStore.transactionDetailsStore(for: txSummary.txInfo)
-            }) {
-              TransactionSummaryView(summary: txSummary, displayAccountCreator: true)
-            }
-            .contextMenu {
-              if !txSummary.txHash.isEmpty {
-                Button(action: {
-                  if let txNetwork = self.networkStore.allChains.first(where: { $0.chainId == txSummary.txInfo.chainId }),
-                     let url = txNetwork.txBlockExplorerLink(txHash: txSummary.txHash, for: txNetwork.coin) {
-                    openWalletURL(url)
-                  }
-                }) {
-                  Label(Strings.Wallet.viewOnBlockExplorer, systemImage: "arrow.up.forward.square")
-                }
-              }
-            }
+        if isShowingBalances.value {
+          VStack(alignment: .trailing) {
+            Text(showBalancePlaceholder ? "0.0000 \(assetDetailStore.assetDetailToken.symbol)" : "\(viewModel.balance) \(assetDetailStore.assetDetailToken.symbol)")
+              .font(.subheadline.weight(.semibold))
+              .foregroundColor(Color(.bravePrimary))
+              .redacted(reason: showBalancePlaceholder ? .placeholder : [])
+              .shimmer(assetDetailStore.isLoadingAccountBalances)
+            Text(showFiatPlaceholder ? "$0.00" : viewModel.fiatBalance)
+              .font(.footnote)
+              .foregroundColor(Color(.braveLabel))
+              .redacted(reason: showFiatPlaceholder ? .placeholder : [])
+              .shimmer(assetDetailStore.isLoadingPrice)
           }
+        } else {
+          Text("****")
+        }
+      }
+      .font(.footnote)
+      .foregroundColor(Color(.secondaryBraveLabel))
+    }
+  }
+
+  @ViewBuilder private var accountsBalanceView: some View {
+    VStack {
+      accountsBalanceHeader
+      
+      if assetDetailStore.accounts.isEmpty {
+        Text(Strings.Wallet.noAccounts)
+        .redacted(reason: assetDetailStore.isLoadingAccountBalances ? .placeholder : [])
+        .shimmer(assetDetailStore.isLoadingAccountBalances)
+        .font(.footnote)
+        .padding()
```

Comment:
I think we should share the empty state with desktop here, similar to transactions tab.

![iOS Accounts](https://github.com/brave/brave-ios/assets/5314553/ad594c83-3c02-4ed2-8dee-2b313a2979e5) | ![iOS Transactions](https://github.com/brave/brave-ios/assets/5314553/d8ad187d-5ee9-49ad-806b-a03813a5a73f)
--|--
![Desktop Accounts](https://github.com/brave/brave-ios/assets/5314553/9c837ea3-4b58-443e-87b3-dc7a40f52f93) | ![Desktop Transactions](https://github.com/brave/brave-ios/assets/5314553/d23da008-00fe-434b-b132-ee81f68fdf68)